signbank/dictionary/templates/dictionary/import_manual_validation_csv_confirmation.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'baselayout.html' %}
+{% load stylesheet %}
+{% load bootstrap3 %}
+{% load i18n %}
+{% block bootstrap3_title %}{% blocktrans %}Confirm Import Manual Validation CSV{% endblocktrans %} | {% endblock %}
+
+{% block content %}
+{% if perms.dictionary.import_csv %}
+    <div>
+      {% if group_row_map %}
+        <h3>{% blocktrans %}ManualValidationAggregations to be added{% endblocktrans %}</h3>
+        {% for group, gloss_count in group_gloss_count.items %}
+          <p>{% blocktrans %}Group: {{ group }} - {{ gloss_count }} glosses{% endblocktrans %}</p>
+        {% endfor %}
+        <form action='{% url "dictionary:confirm_import_manual_validation_csv" %}' method='post'>
+          {% csrf_token %}
+          <input class='btn btn-primary' name='confirm' type='submit' value='{% blocktrans %}Confirm{% endblocktrans %}'>
+          <input class='btn btn-primary' name='cancel' type='submit' value='{% blocktrans %}Cancel{% endblocktrans %}'>
+        </form>
+      {% endif %}
+    </div>
```

Comment:
**question**: should these divs not be mutually exclusive?

Reply:
yes. which means I might as well turn the second one into an else if to the first.